cmd/utils.go

```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/redhat-developer/ocdev/pkg/component"
+	"github.com/redhat-developer/ocdev/pkg/occlient"
+	"github.com/redhat-developer/ocdev/pkg/storage"
+	"github.com/redhat-developer/ocdev/pkg/url"
+)
+
+// printDeleteAppInfo will print things which will be deleted
+func printDeleteAppInfo(client *occlient.Client, appName string) error {
+	componentList, err := component.List(client)
+	if err != nil {
+		return err
+	}
+
+	for _, cmpnt := range componentList {
+		fmt.Println("Component", cmpnt.Name, "will be deleted:")
+		appUrl, err := url.List(client, cmpnt.Name, appName)
+		if err != nil {
+			return err
+		}
+		if len(appUrl) != 0 {
+			fmt.Println("This component is externally exposed, and the URL will be removed")
+		} else {
+			fmt.Println("This component is not externally exposed")
```

Comment:
it might be better to not show anything. To make it consistent with how we display storage.

Reply:
+1